jcommune-view/jcommune-web-controller/src/main/java/org/jtalks/jcommune/web/dto/GroupDto.java

```diff
@@ -27,89 +27,64 @@
  *
  */
 public class GroupDto {
+    private static int DISPLAY_USER_LIMIT= 20;
+
     private long id;
     private String name;
+    private List<UserDto> users;
 
-    /**
-     * Instantiates new GroupDto object based on the Group object
-     * @param group source data for the DTO object
-     */
     public GroupDto(Group group) {
         this.id = group.getId();
         this.name = group.getName();
+        this.users = UserDto.convertToUserDtoList(group.getUsers(), UserDto.BY_NAME_COMPARATOR, DISPLAY_USER_LIMIT);
```

Comment:
yes, for example [Group hibernate-mapping file](https://github.com/jtalks-org/jcommune/blob/master/jcommune-model/src/main/resources/org/jtalks/jcommune/model/entity/Group.hbm.xml) contain named queries.